src/main.ts

```diff
@@ -21,25 +25,69 @@ if (cli.input.length === 0) {
 }
 
 let port = DEFAULT_PORT
+let rest_port = 9999
 if (cli.input.length > 1) {
 	port = Number(cli.input[1])
 	if (isNaN(port)) {
 		cli.showHelp(1)
 	}
+	if (cli.input.length > 2) {
+		rest_port = Number(cli.input[2])
+		if (isNaN(rest_port)) {
+			cli.showHelp(1)
+		}
+	}
 }
 
 console.log('Starting')
 
 const client = new CompanionSatelliteClient({ debug: true })
 const devices = new DeviceManager(client)
+const server = new RestServer(client, rest_port)
```

Comment:
It would be great if setting `rest_port` to 0 would disable this server. I think that at the moment it will result in it choosing a random port instead.